README.md

```diff
@@ -18,16 +29,26 @@ unsee is go installable, so the easiest way is to run:
     go install github.com/cloudflare/unsee
 
 The `unsee` binary will be installed into `$GOPATH/bin` directory.
+Note that this will build latest master version which might not work with your
+Alertmanager version. See
+[supported Alertmanager versions](#supported-alertmanager-versions).
 
 ### Building from source
 
 To clone git repo and build the binary yourself run:
 
     git clone https://github.com/cloudflare/unsee $GOPATH/src/github.com/cloudflare/unsee
     cd $GOPATH/src/github.com/cloudflare/unsee
-    make
 
-`unsee` binary will be compiled in project directory.
+Check [supported Alertmanager versions](#supported-alertmanager-versions) and if
+needed switch to the release branch that supports your Alertmanager version.
+For example to build unsee `0.2.x` run:
+
+    git checkout -t origin/release-0.2
```

Comment:
Do you need `origin/` here? I thought when you set it to track it will do the right thing.